### PR TITLE
QR Code Generator(in real time)

### PR DESCRIPTION
**Pull request by Anuraj-IND**
What I did?
Created a new folder which contain
- A .py file 
- A readme.md file
Also created :
- proper brief about this project in main readme.md file followed by same order 

just make sure to use that hacktoberfestaccepted tag and yea its my first time i tried live frame catching code it was a fun experience !

### CODE REVIEW

**Review by PreyumKr — CHANGES_REQUESTED** [review]
Create a single three hash title and put everything inside that in main readme. Look the readme that you create in preview mode to see how different you write the readme instructions. Also, put the files related to a project inside the project folder. You are using Github GUI or what? Even there its possible to cleanly add files inside a folder. Don't put things outside and keep the readme format as it is from the start. You know how # works for headers right?

**Comment by Anuraj-IND** [comment]
> Create a single three hash title and put everything inside that in main readme. Look the readme that you create in preview mode to see how different you write the readme instructions. Also, put the files related to a project inside the project folder. You are using Github GUI or what? Even there its possible to cleanly add files inside a folder. Don't put things outside and keep the readme format as it is from the start. You know how # works for headers right?

Done

**Comment by Anuraj-IND** [comment]
@PreyumKr done boss check it

**Comment by Anuraj-IND** [comment]
@PreyumKr add hacktoberfestaccepted tag like you did in previous one i need that tag here too
Screenshot 2024-10-18 211609

**Review by PreyumKr — CHANGES_REQUESTED** [review]
### QR Code Reader (In Real Time)

This project demonstrates a simple Python-based QR code scanner that works in real-time using a webcam. It uses the `OpenCV` and `Pyzbar` libraries to detect and decode QR codes live from a video stream and save the results to a text file.

`Overview`

The script captures video input from your webcam, detects QR codes in real-time, and prints the decoded data to the console. After successfully detecting a QR code, the data is appended to a text file and the camera closes automatically.

`Features`

- Real-time QR code detection using a webcam.
- Decodes and saves QR code data to a text file.
- Automatically shuts down the camera after successfully reading a QR code.
- Uses the `OpenCV` library for video capture and the `Pyzbar` library for QR code decoding.
- Supports various QR code formats.

* **Read** the ***readme.md*** given inside the folder and try it out ! ***:-)***


### Header
* point1
* point 2
* point 3
* point 4
*  point 5

Dont add any other header non header or any words in between and keep the points as it is that is using * no - no ` .

**Comment by Anuraj-IND** [comment]
> ### QR Code Reader (In Real Time)
> 
> This project demonstrates a simple Python-based QR code scanner that works in real-time using a webcam. It uses the `OpenCV` and `Pyzbar` libraries to detect and decode QR codes live from a video stream and save the results to a text file.
> 
> `Overview`
> 
> The script captures video input from your webcam, detects QR codes in real-time, and prints the decoded data to the console. After successfully detecting a QR code, the data is appended to a text file and the camera closes automatically.
> 
> `Features`
> 
> - Real-time QR code detection using a webcam.
> - Decodes and saves QR code data to a text file.
> - Automatically shuts down the camera after successfully reading a QR code.
> - Uses the `OpenCV` library for video capture and the `Pyzbar` library for QR code decoding.
> - Supports various QR code formats.
> 
> * **Read** the ***readme.md*** given inside the folder and try it out ! ***:-)***
> 
> 
> ### Header
> * point1
> * point 2
> * point 3
> * point 4
> *  point 5
> 
> Dont add any other header non header or any words in between and keep the points as it is that is using * no - no ` .
> 
> 
> 

I'm having mid term exams now 😭

**Comment by Anuraj-IND** [comment]
done, btw you forgot to add label

**Comment by Anuraj-IND** [comment]
@PreyumKr ig its done now

**Comment by Anuraj-IND** [comment]
response please i need to study now so see if its final nor what

File: README.md
Comment (PreyumKr) [inline README.md:97]: why this space but its ok. Also It looks the file is not synced to the latest readme. Its still showing the old layout but showing only the new lines as newly added so maybe I'll merge and check

**Review by PreyumKr — APPROVED** [review]
There is space and the readme is older than the latest that's there in the main

**Comment by Anuraj-IND** [comment]
@PreyumKr you forgot to add label QwQ  add hacktoberfest-accepted label else itll not count , rn its not counting this pull request

**Comment by PreyumKr** [comment]
Okay I'll add the tag but that's not really a must have. It takes time for a maintainer to approve the PR even with or without the tag. I have all four approved and waiting for review and I don't use the tag for my own PRs.

**Pull request merged**
